cluster/images/etcd/Makefile

```diff
@@ -1,21 +1,47 @@
-.PHONY:	clean build push
+# Build the etcd image
+#
+# Usage:
+# 	[TAG=2.2.2] [REGISTRY=gcr.io/google_containers] [ARCH=amd64] [BASEIMAGE=busybox] make build
 
-IMAGE = etcd
-TAG = 2.2.1
-ETCD_VERSION = 2.2.1
-OUTPUT_DIR = $(IMAGE)-v$(ETCD_VERSION)-linux-amd64
+TAG?=2.2.2
```

Comment:
Can we upgrade versions in a separate PR?  That will make it easier to track (and revert if need be)